main_test.go

```diff
@@ -13,14 +13,14 @@ import (
 )
 
 var (
-	// A valid EC point consists of 64 hex digits.
-	validPoint = "f6414bfccc156551d641260ce403992c5d5b0976aca8a72541fda40e8337d867"
+	// A valid EC point consists of base64.
+	validPoint = "gpfxPFUTfJvKdD6x5G74VD9Bxdb3efsHYJN0d7vu0XE="
```

Comment:
This is a different test point, right?

Reply:
yes

Reply:
All the tests pass for me. This particular point is only used in benchmarks.